giskard_cicd/pipeline/runner.py

```diff
@@ -33,8 +33,8 @@ def run(self, loader_id, **kwargs):
         if scan_config_path is not None:
             with open(scan_config_path) as yaml_f:
                 scan_config = yaml.load(yaml_f, Loader=yaml.Loader)
-            params = dict(scan_config.get("configuration", None))
-            detectors = list(scan_config.get("detectors", None))
+            params = dict(scan_config.get("configuration", {}))
```

Comment:
I think we can also set it to `None` and let [scanner](https://github.com/Giskard-AI/giskard/blob/0c58a9bb29c9b91ac75e774eb92ad54677f3be31/giskard/scanner/scanner.py#L61C9-L61C39) process it:

```
self.params = params or dict()
```

Reply:
This is to avoid the dict() throw NoneType error like this: 
```
    params = dict(scan_config.get("configuration", None))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
```